app/scripts/modules/core/src/api/ApiService.ts

```diff
@@ -164,9 +164,9 @@ export class API {
   private static init(urls: string[]) {
```

Comment:
you'd think the `toString()` on these was optional, but there were a few tests that found callers that passing in url objects instead of Strings

Reply:
We should fix those tests and not call `toString` here. The app won't work if `gateUrl` is not a string; this kind of implies it could.

Reply:
Okay, we talked through this, Cam is half right

Reply:
I'm a glass-is-half-right kind of guy

Reply:
> Okay, we talked through this, Cam is half right

What was the secret discussion?

Reply:
lol that was so long ago